dps/utils/Unfolding.py

```diff
@@ -88,6 +88,53 @@ def refold(self):
         else:
             print("Data has not been unfolded. You cannot refold something that hasn't been first unfolded")
         return self.refolded_data
+    def get_covariance_matrix(self):
+        '''
+        Get the covariance matrix from all contributions
+        https://root.cern.ch/doc/master/classTUnfoldDensity.html#a7f9335973b3c520e2a4311d2dd6f5579
```

Comment:
Is this for all contributions including systematic, or just statistical?  I assume statistical if it's coming straight from TUnfold.